nodes/widgets/ui_control.js

```diff
@@ -18,8 +18,8 @@ module.exports = function (RED) {
             })
         }
 
-        function emit (payload) {
```

Comment:
Sending just the `payload` meant we could never perform socket connection constraint checks